Examples/Run/HepMC3/CMakeLists.txt

```diff
@@ -4,9 +4,8 @@ add_executable(
 target_link_libraries(
   ActsExampleHepMC3
   PRIVATE
-    ActsCore ActsExamplesCommon
-    ActsExamplesFramework ActsExamplesIoHepMC3
-    Boost::program_options)
+    ActsCore ActsFatras 
+    ActsExamplesCommon ActsExamplesFramework ActsExamplesIoHepMC3)
```

Comment:
program options not needed here?

Reply:
Should already be provided through `ActsExamplesCommon`.